src/test/kotlin/dev/fakek/fakes/FakeCompanyTest.kt

```diff
@@ -0,0 +1,54 @@
+package dev.fakek.fakes
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import strikt.api.expect
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+
+class FakeCompanyTest {
+
+    private val bs = "fake everthing"
+    private val buzzword = "Fake-focused"
+    private val catchPhrase = "fake until you make it"
+    private val industry = "Fake Pproduct Services"
+    private val logo = "https://pigment.github.com/fake-logos/logos/medium/color/5.png"
+    private val name = "Faker-Corporate"
+    private val profession = "Faker"
+    private val suffix = "Group"
+
+    private val fakerCompany: FakerCompany = mockk {
+        every { bs() } returns bs
+        every { buzzword() } returns buzzword
+        every { catchPhrase() } returns catchPhrase
+        every { industry() } returns industry
+        every { logo() } returns logo
+        every { name() } returns name
+        every { profession() } returns profession
+        every { suffix() } returns suffix
+    }
+
+    private val url = "www.fake-url.com.fk"
+
+    private val fakerUrl = FakeUrl(url)
+
+    private fun fakeCompany() = FakeCompany(fakerCompany, fakerUrl)
+
+    @Test
+    fun `given a FakerCompany and FakerUrl when creating a FakeCompany then the correct data should be set`() {
+        val subject = fakeCompany()
+
+        expectThat(subject) {
```

Comment:
I haven't had a chance to use `expectThat` in this way yet. If the first `get` fails will the other lines still execute? I've tried to keep tests to a single assertion so if one fails the other assertions can still be executed, this might be an outdated thought though if `expectThat` will evaluate each line independently.

Reply:
yes they will.

You can check the usage in here https://strikt.io/wiki/traversing-subjects/

I got your intention in the tests you created and I looked after to see if this assertion lib offered something along those lines to avoid create different samples for each field/method

Reply:
Awesome great to hear, thank you!